src/main/webapp/app/exercises/shared/participation/participation.component.html

```diff
@@ -38,6 +38,14 @@ <h2>
             *ngIf="exercise?.isAtLeastInstructor && exercise?.type === ExerciseType.PROGRAMMING"
             [exercise]="exercise"
         ></jhi-programming-exercise-instructor-submission-state>
+        <div class="p-2">
+            <a class="btn btn-info" *ngIf="exercise?.isAtLeastTutor" [routerLink]="getScoresRoute(exercise)" style="display: flex; justify-content: center; align-items: center">
```

Comment:
```suggestion
            <a class="btn btn-info" *ngIf="exercise?.isAtLeastTutor" [routerLink]="getScoresRoute(exercise)">
```
It seems to still be centered even with the inline style removed.